style/swift/sample.swift

```diff
@@ -31,16 +31,19 @@ var maybe: Bool?
 
 // Use if-let syntax to unwrap optionals
 if let definitely = maybe {
-    println("This is \(definitely) here")
+    print("This is \(definitely) here")
 }
 
 // If the API you are using has implicit unwrapping you should still use if-let
 func someUnauditedAPI(thing: String!) {
     if let string = thing {
-        println(string)
+        print(string)
     }
 }
 
+// You can also use the `?` postfix operator to chain possible execution
+delegate?.doSomething()
```

Comment:
Oh, I see. I'd be more in favor of removing the weirder non-guidelines that boil down to "Use language features" in a separate PR. I don't think this kind of thing is going to scale. The guideline _should_ be "Avoid force-unwrapping optionals" so maybe we just change it to that?